haas/commands/admin.py

```diff
@@ -0,0 +1,16 @@
+from haas import config, model
+from haas.commands import db
+from haas.flaskapp import app
+from flask.ext.script import Manager
+
+manager = Manager(app)
+manager.add_command('db', db.command)
+
+
+def main():
+    """Entrypoint for the haas-admin command."""
+    config.load()
+    config.configure_logging()
+    config.load_extensions()
```

Comment:
I feel like we do this mantra in several places now (`config.load(); config.configure_logging(); config.load_extensions()`), including cli.py and haas.wsgi, in violation of [DRY](https://en.wikipedia.org/wiki/Don't_repeat_yourself).

What do you think about placing these into a single call in config.py?